build_library/prod_image_util.sh

```diff
@@ -247,3 +247,42 @@ create_prod_tar() {
   sudo losetup --detach "${lodev}"
   upload_image "${container}"
 }
+
+DEFAULT_SYSEXTS=(
```

Comment:
Can we call this EXTRA_SYSEXTS as these won't be installed by default? I imagine that we could have `DEFAULT_SYSEXTS` in the future for things that are pre-enabled.

Reply:
will do

Reply:
I think this listing should be in a separate file (see `ci-automation/base-sysexts.sh`), so in future an automation could obtain a list of extra sysexts (for generating image changes or deciding if a package update affects some sysext). With this done, I think that the list of extra sysexts should be passed directly to `create_prod_sysexts` function then.

Reply:
In a different file for sourcing? That could make sense, but then it shouldn't be in `ci-automation` and instead somewhere in `build_library`, so that it can be shared with `build_image`. Which sysexts we build shouldn't be a CLI parameter.

Reply:
The file in ci-automation was an example. It's fine if it's under `build_library`.

Reply:
moved to `./build_library/extra_sysexts.sh`